src/lib/utils.js

```diff
@@ -0,0 +1,37 @@
+import { MissionUtils } from '@woowacourse/mission-utils';
+
+export const intersection = (arrayA, arrayB) =>
+  arrayA.filter((itemA) => arrayB.includes(itemA));
+
+export const getIsNumeric = (value) => !Number.isNaN(Number(value));
+
+export const getIsThousandUnit = (value) => Number.isInteger(value / 1000);
+
+export const getIsPositive = (value) => Number(value) > 0;
+
+export const getIsArrayLengthMatch = (array, length) => array.length === length;
+
+export const getIsAllItemsNumeric = (array) =>
+  array.every((item) => getIsNumeric(item));
+
+export const getIsBetweenNumbers = (value, min, max) =>
+  value >= min && value <= max;
+
+export const getIsAllItemsBetweenNumbers = (array, min, max) =>
+  array.every((item) => getIsBetweenNumbers(item, min, max));
+
+export const getIsAllItemsUnique = (array) =>
+  array.length === new Set(array).size;
+
+export const retryWhileCatchedError = async (callbackFunction) => {
+  try {
+    return await callbackFunction();
+  } catch (error) {
+    MissionUtils.Console.print(error.message);
+
+    const retried = await retryWhileCatchedError(callbackFunction);
+    return retried;
```

Comment:
```suggestion
    return await retryWhileCatchedError(callbackFunction);
```
요렇게 축약할 수 있을 것 같아요!
따로 변수에 담아두신 이유가 있으신가용?

Reply:
변수에 담아 의미를 부여하기 위해 담았습니다.

그런데 `retried`는 약간 에매한 네이밍 같네요 😢

Reply:
의미부여의 의도였다면, 축약하셔도 충분히 의미가 잘 전달되지 않나요? 저는 개인적으로 축약하지 않아도 충분히 의미 전달이 되면 축약을 안하는 것 같아요! 😄